### PR TITLE
Two refactorings

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

**Comment by mulkieran** [comment]
@agrover @trgill review?

**Comment by mulkieran** [comment]
Added a small cleanup.

File: src/engine/strat_engine/blockdev.rs [hunk 1]
```diff
@@ -102,7 +102,7 @@ pub fn initialize(pool_uuid: &PoolUuid,
                   devices: BTreeSet<Device>,
                   mda_size: Sectors,
                   force: bool)
-                  -> EngineResult<BTreeMap<PathBuf, BlockDev>> {
+                  -> EngineResult<Vec<BlockDev>> {
```
Comment (agrover) [inline src/engine/strat_engine/blockdev.rs]: How about a `*Set`? Blockdevs should have no order.

**Comment by mulkieran** [comment]
Sticking with Vec type for initialize result for now.

**Pull request merged**